integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -75,10 +77,18 @@ def _partition_items(self, channel_metadata_item_map):
                 if diff(channel_metadata, transmitted_item.channel_metadata):
                     items_to_update[content_id] = channel_metadata
                     items_catalog_updated_times[content_id] = item.content_last_changed
+                else:
+                    skipped_items.append(content_id)
             else:
                 items_to_create[content_id] = channel_metadata
+                content_catalog_map[content_id] = item.enterprise_customer_catalog_uuid
                 items_catalog_updated_times[content_id] = item.content_last_changed
 
+        if skipped_items:
```

Comment:
nit: sorry for last minute feedback. All the other items are actual objects/dicts. This is actually a array of scalars (content+ids). Shall we just name it like that `skipped_content_ids`?